### DISCUSSION TITLE
Visual Servoing

### DISCUSSION THREAD

**Discussion by ProgrammingAlessandro** [UniversalRobots/Universal_Robots_ROS2_Driver#1014] [Q&A] [answered]
Hi is there any way to do visual Servoing with ROS2 and this UR pkg?

I know in the ROS1 i can use the [twist_controller](https://wiki.ros.org/twist_controller), but as far as I know this controller does not (yet?) exist for this UR Repo?

Thanks in advance

**Comment by fmauch** [comment, marked as answer]
This does not exist in ROS 2 (yet).
